utility/doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contribution guide
+
+- For reference, [here is the detailed version](https://github.com/pokt-network/pocket/blob/main/utility/doc/E2E_FEATURE_PATH_TEMPLATE.md#e2e-feature-implementation)
+  
+- This doc assumes you know exactly what to work on, i.e. you are assigned [an issue with details hashed out](https://github.com/pokt-network/pocket/issues/754).
+
+- This only applies to the `utility` module.
+
+# Steps
```

Comment:
Instead of a separate contribution guide, what if we just add a `tl;dr` section to the E2E future list section?

Reply:
SGTM, will update the PR shortly.